### DISCUSSION TITLE
Conditional branch 'Has sprite'

### DISCUSSION THREAD

**Discussion by Gibier1** [AscensionGameDev/Intersect-Engine#1786] [Ideas]
The engine allow to create classes with many sprite. You can also change sprite and classes IG but changing classes doesn't take the player's sprite in consideration. In my game I'll like that at a certain age the player grow up to a bigger sprite. 

A conditional branch for 'Has sprite' could also do many of things like pet talk if you transform into an animal.

**Comment by SilverShardOnline** [comment]
I've made a similar suggestion also.

This allows for many systems: Aging, Mounts, Sailing, Haircuts/Skins, Cut Scene actions, etc.

The engine already knows this info, so hopefully would be an easy enhancement.

**Comment by WeylonSantana** [comment]
duplicate https://github.com/AscensionGameDev/Intersect-Engine/discussions/1703
